utils/tolerance.stanza

```diff
@@ -0,0 +1,60 @@
+#use-added-syntax(jitx)
+defpackage ocdb/tolerance:
+  import core
+  import collections
+  import math
+
+; A generic value with some tolerance. May be offcenter (upper tolerance != lower tolerance)
+public pcb-struct ocdb/tolerance/Toleranced:
+  tol+:Double,
+  tol-:Double,
+  typ: Double,
+
+defmethod print (o:OutputStream, t:Toleranced):
+  println(o, "Toleranced(%_ <= %_ <= %_)" % [min-value(t), typ(t), max-value(t)])
+
+; Create a tolerance value with the same tolerance in both directions
+public defn Toleranced (value:Double, tolerance:Double) -> Toleranced:
```

Comment:
Use the same convention for all the `Toleranced` constructors. 

Either they are called `Toleranced` to follow standard Stanza convention, or they have their own short names (e.g. `tol%`).

Reply:
Done